ipn/ipnlocal/local.go

```diff
@@ -2306,6 +2333,7 @@ func (b *LocalBackend) getNewControlClientFuncLocked() clientGen {
 
 // initOnce is called on the first call to [LocalBackend.Start].
 func (b *LocalBackend) initOnce() {
+	go b.consumeEventbusTopics()
```

Comment:
We try to avoid fire-and-forget goroutines. At the very least, let's avoid starting one directly and do this instead:
```suggestion
	b.goTracker.Go(b.consumeEventbusTopics)
```
But also, we should consider waiting for `(*LocalBackend).consumeEventbusTopics()` to return in `(*LocalBackend).Shutdown()`.

See also: 07aae18bca5a26e459b3d3f906f4c099c3f75df0

Reply:
To clarify: using `b.goTracker.Go` helps with flaky tests, such as when a goroutine started during a test keeps running and logs after the test has completed (e.g., #15938). But it's not a replacement for a proper shutdown wait, which we likely want for `consumeEventbusTopics`.